apps/bixarena/tools/README.md

```diff
@@ -1,3 +1,78 @@
-# bixarena-tools
+# BixArena Tools
 
-Project description here.
+Tools for Bradley-Terry evaluation and ranking in the BixArena project.
```

Comment:
BT evaluation is only one tool of the `bixarena-tools` project. It should be described in a section of the project README instead of taking it over. Consider placing this tool in its own Python module.